rasa/core/training/interactive.py

```diff
@@ -579,6 +579,18 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
     return slot_strings
 
 
+async def _retry_on_error(func: Callable, export_path: Text, *args, **kwargs):
+    while True:
```

Comment:
`while True` seems dangerous - how about giving the user 1 chance or a limited number (e.g. 3) tries to fix the issue?

Reply:
I am not sure what's dangerous about it. Successful exports return. Everything which is not an OSError raises and all exceptions outside of the export function raise as well.

Imo the user should be given as many tries as needed to fix the problem. Some file errors can take a few tries to solve.

But if really necessary I can change it.